portfolio/src/main/webapp/script.js

```diff
@@ -18,25 +18,38 @@
 */
 function scrollToID(id) {
   var y = document.getElementById(id).offsetTop - document.getElementById("header").offsetHeight;
-  //console.log("height of header " + document.getElementById("header").offsetHeight);
   console.log("top of elm " + document.getElementById(id).offsetTop);
-  //console.log("jumping to " + y);
   window.scrollTo({top: y, behavior: 'smooth'});
 }
 
 window.onscroll = function() {
-    highlightTab();
+  highlightTab();
 };
 
 /*
 * Highlights current tab on nav bar
 */
 function highlightTab() {
-    if (window.pageYOffset < document.getElementById("project").offsetTop - document.getElementById("header").offsetHeight) {
-        document.getElementById("about-btn").classList.add("curr-selection");
-        document.getElementById("project-btn").classList.remove("curr-selection");
-    } else {
-        document.getElementById("about-btn").classList.remove("curr-selection");
-        document.getElementById("project-btn").classList.add("curr-selection");
-    }
+  if (window.pageYOffset < document.getElementById("project").offsetTop - document.getElementById("header").offsetHeight) {
+    document.getElementById("about-btn").classList.add("curr-selection");
+    document.getElementById("project-btn").classList.remove("curr-selection");
+  } else {
+    document.getElementById("about-btn").classList.remove("curr-selection");
+    document.getElementById("project-btn").classList.add("curr-selection");
+  }
 }
+
+function getDataFromServlet() {
+  fetch('/data').then(response => response.text()).then((quote) => {
+    document.getElementById('data-container').innerText = quote;
+  });
+}
+
+function getJson() {
+    (fetch('/data').then(response => response.json()).then((list) => {
+        console.log(list);
+        for (let i = 0; i < list.length; i++) {
```

Comment:
Since this is specific to comments I'd rename `list` to `comments`. 

Also consider changing the for loop structure to be something like this... Unless you really want to keep IE around :) 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of
```
for(const comment of comments) {
  document.getElementById("data-container").innerText += comment + "\n";
}